presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -41,27 +50,45 @@ Array Functions
 
     Concatenates the elements of the given array using the delimiter and an optional string to replace nulls.
 
+        SELECT array_join(ARRAY[1,1,2], '|'); -- 1|1|2
+        SELECT array_join(ARRAY[1,1,null], '|', 'NaN'); -- 1|1|NaN
+
 .. function:: array_max(x) -> x
 
     Returns the maximum value of input array.
 
+        SELECT array_max(ARRAY[-1,1,2]); -- 2
+        SELECT array_max(ARRAY['a','b','c']); -- c
+
 .. function:: array_min(x) -> x
 
     Returns the minimum value of input array.
 
+        SELECT array_min(ARRAY[-1,1,2]); -- -1
+        SELECT array_min(ARRAY['a','b','c']); -- a
+
 .. function:: array_position(x, element) -> bigint
 
     Returns the position of the first occurrence of the ``element`` in array ``x`` (or 0 if not found).
 
+        SELECT array_position(ARRAY[10,20,30,40,50],10); -- 1
+        SELECT array_position(ARRAY[10,20,30,40,50],1); -- 0
```

Comment:
- Have some examples where the array is out-of-order
- Example with array of VARCHAR

Besides array contains null values, also show what will happen:
- when the `element` to be found is null